golem_messages/message/base.py

```diff
@@ -56,7 +56,7 @@ class Message():
 
     __slots__ = ['timestamp', 'encrypted', 'sig', '_raw']
 
-    HDR_LEN = 11
+    HDR_LEN = struct.calcsize('!HQ?')

```

Comment:
Not sure how often this is used, but maybe the string can be a const like variable too?